cmd/build.go

```diff
@@ -55,46 +48,52 @@ func Build(ctx context.Context) *cobra.Command {
 			if err := utils.CheckIfDirectory(path); err != nil {
 				return fmt.Errorf("invalid build context: %s", err.Error())
 			}
+			options.Path = path
 
-			if file == "" {
-				file = filepath.Join(path, "Dockerfile")
+			if options.File == "" {
+				options.File = filepath.Join(path, "Dockerfile")
 			}
 
-			if err := utils.CheckIfRegularFile(file); err != nil {
+			if err := utils.CheckIfRegularFile(options.File); err != nil {
 				return fmt.Errorf("invalid Dockerfile: %s", err.Error())
 			}
 
 			buildKitHost, isOktetoCluster, err := build.GetBuildKitHost()
 			if err != nil {
 				return err
 			}
-			log.Information("Running your build in %s...", buildKitHost)
+			if buildKitHost == "" || options.BuildLocally {
+				log.Information("Running your build in docker...")
```

Comment:
what would happen if the user doesn't have a docker daemon running? (or is using something else like podman, ko, etc...)

Reply:
Also, what if the docker daemon is in a different machine? are we support that as well?

Reply:
If the user does not have docker running it will fail when trying to reach the api endpoint, the same as docker does.
At the moment we are not supporting that the docker daemon is in another machine. I think this is a first step to make `okteto stack/up/push` be available on most clusters

Reply:
cool, let's make sure we have a clear error for that case  then?